README.md

```diff
@@ -135,6 +135,15 @@ Import the 3 MIDI files thus generated into your favorite music creation softwar
 
 [Simple kick drum with bass and some hats generated by Scribbletune](https://soundcloud.com/walmik/loop) 
 
+### Changing Middle C
+Sometimes Middle C is not C4, but a C in a different octave. To set the octave for middle C use
+```js
+scribble.setMiddle(octave)
```

Comment:
Rename the method to `setMiddleC` and give a full example of how to use this feature where a clip is created with a new middle C